packages/component-library/stories/HorizontalBarChart.story.js

```diff
@@ -2,122 +2,237 @@ import React from "react";
 /* eslint-disable import/no-extraneous-dependencies */
 import { storiesOf } from "@storybook/react";
 import { checkA11y } from "@storybook/addon-a11y";
-import { withKnobs, text, object } from "@storybook/addon-knobs";
+import {
+  withKnobs,
+  text,
+  object,
+  boolean,
+  optionsKnob as options
+} from "@storybook/addon-knobs";
 import { HorizontalBarChart, civicFormat } from "../src";
+import notes from "./horizontalBarChart.notes.md";
 
-const sampleData = [
-  { sortOrder: 1, population: 2000, label: "Labrador Retriever" },
-  { sortOrder: 2, population: 8000, label: "Standard Poodle" },
-  { sortOrder: 3, population: 6000, label: "French Bulldog" },
-  { sortOrder: 4, population: 3000, label: "Afghan Hound" },
-  { sortOrder: 5, population: 1000, label: "Jack Russell Terrier" }
-];
-
-const sampleMinimalistData = [
-  { sortOrder: 1, population: 2000, label: "Labrador Retriever" },
-  { sortOrder: 2, population: 8000, label: "Standard Poodle" }
-];
-
-const sampleUnsortedData = [
-  { population: 2000, label: "Labrador Retriever" },
-  { population: 8000, label: "Standard Poodle" },
-  { population: 6000, label: "French Bulldog" },
-  { population: 3000, label: "Afghan Hound" },
-  { population: 1000, label: "Jack Russell Terrier" }
-];
+const GROUP_IDS = {
+  LABELS: "Labels",
+  DATA: "Data",
+  CUSTOM: "Custom"
+};
+const getKeyNames = obj => {
```

Comment:
Since this is reused in BarChart, HorizontalBarChart, and will be used in other stories, extract to shared.js helper file and import rather than duplicating.

Reply:
I will also extract getKeyNames from the BarChart story and import from shared.js.  (In a new branch.)